src/pages/storage/storage-list.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <Page title="Storage Settings" :breadcrumb-links="[]" :error=errors>
+    <template #header-right>
+      <router-link to="/storages/new" v-slot="{ href, route, navigate}" tag="">
+        <b-button variant="primary" @click="navigate">Create New Storage</b-button>
+      </router-link>
+    </template>
+    <div class='w-100 p-2'>
+      <ul>
+        <li v-for="storage in storages" :key="storage.storageId">
+          <div style="display: flex; flex-direction: row;">
+            <div>
+              <b-button v-if="storageExpanded[storage.storageId]" variant="link" size="sm" v-b-tooltip.hover title="Collapse" v-on:click="onClickExpandOrCollapse(storage.storageId)">
+                <b-icon icon="chevron-down"></b-icon>
+              </b-button>
+              <b-button v-else variant="link" size="sm" v-b-tooltip.hover title="View Storage Preferences" v-on:click="onClickExpandOrCollapse(storage.storageId)">
+                <b-icon icon="chevron-right"></b-icon>
+              </b-button>
+            </div>
+            <div style="flex: 1;">
+              <span>{{ storage.hostName}}:{{storage.port}} <span style="font-size:12px">({{ storage.storageId }})</span></span>
+            </div>
+            <div style="margin-right: 20rem">
+              <router-link 
+                :to="{name: 'storage-preferences-new', query:{storageId: `${storage.storageId}`}}" 
+                v-slot="{ href, route, navigate}" tag="">
+                <b-button variant="link" size="sm" @click="navigate" v-b-tooltip.hover title="Create Storage Preferences">
+                  <b-icon icon="folder-plus"></b-icon>
+                </b-button>
+              </router-link>
+            </div>
+          </div>
+          <div v-if="!storageExpanded[storage.storageId]" class="w-100 p-2">
+            <table-overlay-info :rows="5" :columns="5" :data="storagePreferencesByStorageId[storage.storageId]">
```

Comment:
There's a slot you could override to customize the empty view. 
https://github.com/apache/airavata-custos-portal/blob/master/src/lib/components/overlay/table-overlay-info.vue#L5

The following may be not suitable.
<img width="1179" alt="Screenshot 2021-07-27 at 14 06 50" src="https://user-images.githubusercontent.com/10680296/127205776-d4a7c229-563c-4a04-aab9-ea11332c41fb.png">

Reply:
Overrides the customized empty view.